chef/cookbooks/bind9/recipes/default.rb

```diff
@@ -403,6 +403,7 @@ def make_zone(zone)
   command "nscd -i hosts"
   action :nothing
   subscribes :run, "template[/etc/bind/db.#{node[:dns][:domain]}]"
+  only_if { File.exist?("/var/run/nscd/nscd.pid") }
```

Comment:
just checked on Debian-7.9 and 8.3 - it also uses /var/run/nscd/nscd.pid so that has pretty good chances of working on most relevant platforms.